NDTensors/src/tensor/tensor.jl

```diff
@@ -184,6 +188,8 @@ setstorage(T, nstore) = tensor(nstore, inds(T))
 
 setinds(T, ninds) = tensor(storage(T), ninds)
 
+isemptystorage(t::Tensor) = datatype(t) isa NDTensors.Zeros
```

Comment:
I think it would make sense to rename this, since there won't be an `EmptyStorage` type going forward. Maybe `iszerodata`?